src/pygeoprocessing/routing/routing.pyx

```diff
@@ -3813,11 +3827,26 @@ def calculate_subwatershed_boundary(
             raster
         strahler_stream_vector_path (str): path to stream segment vector
         target_watershed_boundary_vector_path (str): path to created vector
-            of stringline for watershed boundaries.
+            of stringline for watershed boundaries. Contains the fields:
+            "stream_id": this is the stream ID from the
+                ``strahler_stream_vector_path`` that corresponds to this
+                subwatershed.
+            "terminated_early": if set to 1 this watershed generation was
+                terminated before it could be complete. This value should
+                always be 0 unless something is wrong as a software bug
+                or some degenerate case of data.
+            "outlet_x", "outlet_y": this is the x/y coordinate in raster
+                space of the outlet of the watershed. It can be useful when
+                determining other properties about the watershed when indexed
+                with underlying raster data that created the streams in
+                ``strahler_stream_vector_path``.

```

Comment:
I think this will be a bit malformed in the rendered RST.  Maybe a bulleted list (like in the docstring for `extract_strahler_streams_d8`) would work better?

Reply:
Good idea, changed to bulleted list in e704e04

Reply:
p.s. in schools these days they say "jot dots" instead of "bullet list" for the obvious tragic implications. I wonder when this will roll around to our world. so... just a heads up that'll happen someday.

Reply:
Well that is interesting ... I'll try to use "jot dots" instead.  Thanks for the heads up!